pkg/dockerfile/dockerfile.go

```diff
@@ -40,6 +42,15 @@ func Parse(b []byte) ([]instructions.Stage, error) {
 	return stages, err
 }
 
+func Validate(stages []instructions.Stage, target string) error {
```

Comment:
nit: this makes it sound like you're going to validate the whole Dockerfile, when all we're really checking is that the target exists. Maybe rename it to that?

Reply:
done